pymeasure/instruments/instrument.py

```diff
@@ -101,6 +102,35 @@ def id(self):
         else:
             raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
 
+    @property
+    def quick_properties_list(self):
+        return self.quick_props_list
+
+    @quick_properties_list.setter
+    def quick_properties_list(self, prop_list):
+        if type(prop_list) is str:
+            self.quick_props_list = [prop_list]
+        elif type(prop_list) is list:
+            self.quick_props_list = prop_list
+        else:
+            raise TypeError("Expected input of type str or list")
+
+    @property
+    def quick_properties(self):
+        prop_dict = {}
+        for prop_name in self.quick_props_list:
+            prop_dict[prop_name] = getattr(self, prop_name)
```

Comment:
I don't know if you're a fan of comprehensions, but 
```
prop_dict = {prop_name: getattr(self, prop_name) for prop_name in self.quick_props_list}
```
should be a more concise way of generating this.